src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -264,29 +294,32 @@ void convertRequireToImportStatements(Node n, String fullLocalName, String requi
 
     if (module.providesObjectChildren.get(requiredNamespace).size() > 0) {
```

Comment:
should this be an else if? It doesn't look like it makes sense to hit both code paths.